azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKEventHandler.kt

```diff
@@ -526,11 +542,19 @@ internal class CallingSDKEventHandler(
 
         isSpeakingChangedListenerMap[id] = addOnIsSpeakingChangedEvent
         addedParticipant.addOnIsSpeakingChangedListener(isSpeakingChangedListenerMap[id])
+        val addOnIsDisplayNameChangedEvent =
+            PropertyChangedListener {
+                remoteParticipantsInfoModelMap[id]?.displayName =
+                    remoteParticipantsCacheMap[id]!!.displayName
+                onRemoteParticipantPropertyChange(id)
+            }
+        isDisplayNameChangedListenerMap[id] = addOnIsDisplayNameChangedEvent
+        addedParticipant.addOnDisplayNameChangedListener(addOnIsDisplayNameChangedEvent)
     }
 
     private fun onRemoteParticipantUpdated() {
         val state = call?.state
-        if (state == CallState.CONNECTED) {
+        if (state == CallState.CONNECTED || state == CallState.REMOTE_HOLD) {
```

Comment:
inversion is useful here, state = call?.state so it could be null.

null == CallStateConnected, NPE.

if (CONNECTED == state || ...)

Safer

Reply:
As per discussion we will fix for GA.